pages/txs/computor/[hash].tsx

```diff
@@ -0,0 +1,20 @@
+import type { NextPage } from 'next';
+import dynamic from 'next/dynamic';
+import React from 'react';
+
+import type { Props } from 'nextjs/getServerSideProps';
+import PageNextJs from 'nextjs/PageNextJs';
+
+const ComputorTxs = dynamic(() => import('ui/pages/ComputorTxs'), { ssr: false });
+
+const Page: NextPage<Props> = (props: Props) => {
+  return (
+    <PageNextJs pathname="/txs/computor/[hash]" query={ props }>
+      <ComputorTxs/>
+    </PageNextJs>
+  );
+};
+
+export default Page;
+
+export { base as getServerSideProps } from 'nextjs/getServerSideProps';
```

Comment:
I think there should be an env variable or some other way to return 404 error for chains that shouldn't have this page.